triggerGHR/README.md

```diff
@@ -0,0 +1,69 @@
+# triggerGHR Script
+
+The script “triggerGHR.sh” component is created to automate the process of triggering GHR through ImpactRP API by mentioning the failure scenarios based on the NHC test. 
+
+## Prerequisite
+
+1) If the user wants to use NHC and report failure to GHR, they must be onboarded to both Azure health check (NHC) and Azure Impact Reporting (GHR) services. 

```

Comment:
How does someone get onboarded to GHR? I think we need to be very clear here because this makes it sound like anyone can just do it but at the same time, we aren't providing any way to get onboarded. 

If there is an official process that should be stated here if not maybe we should state that this is a limited offering.

Reply:
Okay, will think of adding more description to readme file.